feature: add news section and reformat home page (#4)
llm4code/llm4code.github.io: index.md

  </picture>
</p>

> We are pleased to announce that the 1st International Workshop on Large Language Models for Code (**LLM4Code**) will take place in Lisbon, co-located with [ICSE 2024](https://conf.researchr.org/home/icse-2024)! The submission deadline is **Dec 7th**, 2023. The link to submit is [here](https://llm4code2024.hotcrp.com/).

Reviewer: Let's remove this line and embed this into news.

> We are pleased to announce that the 1st International Workshop on Large Language Models for Code (**LLM4Code**) will take place in Lisbon, co-located with [ICSE 2024](https://conf.researchr.org/home/icse-2024)! The submission deadline is **Dec 7th**, 2023. The link to submit is [here](https://llm4code2024.hotcrp.com/).

Reviewer 2: ```suggestion
We are pleased to announce that the 1st International Workshop on Large Language Models for Code (**LLM4Code**) will take place in Lisbon, co-located with [ICSE 2024](https://conf.researchr.org/home/icse-2024)! The submission deadline is **Dec 7th**, 2023. You can submit through [this link](https://llm4code2024.hotcrp.com/).
```

## News

> 

Reviewer: ```suggestion
- **Oct 11, 2023** Our [HotCRP site](https://llm4code2024.hotcrp.com/) is ready for your submissions!
```

> - **Oct 11, 2023** Our [HotCRP site](https://llm4code2024.hotcrp.com/) is ready for your submissions!

Reviewer: add some quick deadline info

Author: Added

> - **Oct 11, 2023** We have launched our Twitter account ([@llm4code](https://twitter.com/llm4code)) with official hashtag **#LLM4Code**!

Reviewer: ```suggestion
- **Oct 11, 2023** Follow us on Twitter/X ([@llm4code](https://twitter.com/llm4code)) to hear timely updates!
```
